tests/camera/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+add_executable(CameraTest CameraTest.cc)
+target_link_libraries(CameraTest
+  gtest_main
+  gz-sim${GZ_SIM_VER}::gz-sim${GZ_SIM_VER}
+  gz-plugin${GZ_PLUGIN_VER}::gz-plugin${GZ_PLUGIN_VER}
+  ${YARP_LIBRARIES}
+)
+add_test(NAME CameraTest COMMAND CameraTest)
+
+set(_env_vars)
+list(APPEND _env_vars "GZ_SIM_SYSTEM_PLUGIN_PATH=$<TARGET_FILE_DIR:GazeboYarpCamera>")
+list(APPEND _env_vars "LIBGL_ALWAYS_SOFTWARE=1")
```

Comment:
I saw that you added this in the Laser tests as well. However, this should only be necessary if you are using WSL2, not in general. Furthermore, it should not be necessary anymore after the release of gz-renderng 7.4.1 , see https://github.com/gazebosim/gz-sim/issues/920#issuecomment-1683223479 . However, I will open a separate issue for this.